ReactiveExample/TerminalScheduler.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Reactive.Concurrency;
+using System.Reactive.Disposables;
+using Terminal.Gui;
+
+namespace ReactiveExample {
+	public class TerminalScheduler : LocalScheduler {
```

Comment:
I think you are suggesting this could be built-in to `Terminal.Gui`? How would existing code (e.g. the Progress Bar scenario) be updated to use this?

Reply:
If we add this scheduler to `Terminal.Gui`, the change will introduce a dependency on `System.Reactive`, and this may result in folks complaining about extra `.dll` files they don't use etc. Probably worth creating a separate package named `Terminal.Gui.ReactiveUI` for this

Reply:
That makes tons of sense. Looking forward to that PR.